openSUSE/openSUSE-release-tools
# debug

- Run TTM for openSUSE:Leap:15.4:ARM (armv7)
- Add 15.4/ARM Images totestmanager
- pkglistgen: Handle loops in project path expansion
- Update MicroOS IBS project (MicroOS52) in botmaster
- Add a dashboard for Leap
- Dashboard: do not display containerfile values for Leap
- dist: include project-installcheck.py again
- spec-cleaner imposed cleanups
- gocd: Drop 15.2
- gocd: Monitor KDE:Medias as well
- gocd: Update link to Leap-15.3.aarch64-libvirt_aarch64.json
- gocd: Add Leap 15.4 vagrant boxes
- skippkg-finder: add a supplement list for adding package to the skiplist
- factory-package-news.py: Don't rely on changelogtime equality
- Check whether the FTP trees contain OSRT:ProductVersion before releasing
- ci: We don't use coveralls anymore
- Include :Staging in :adi images and containerfile repo paths
- ToolBase: use --osc-debug for osc's config value, not the tools own debug parameter
